parent/pom.xml

```diff
@@ -1395,42 +1395,32 @@
             </build>
         </profile>
         <profile>
+            <id>Performance</id>
+            <properties>
+                <includedTestGroups>Performance</includedTestGroups>
+                <excludedTestGroups>Live,WIP,Broken</excludedTestGroups>
+            </properties>
+        </profile>
+        <profile>
             <id>Acceptance</id>
             <properties>
                 <includedTestGroups>Acceptance</includedTestGroups>
                 <excludedTestGroups>Integration,Live,WIP,Broken</excludedTestGroups>
```

Comment:
Why exclude `Performance` in the other groups but not here?
To have the tests which are annotated with two groups to run the of of the groups should not be excluded from the profile of the other one. But then why make `Performance` special?
I think we should just exclude `Broken` and let mixed tests execute in either of the groups.